docs/source/06_nodes_and_pipelines/04_micro_packaging.md

```diff
@@ -0,0 +1,124 @@
+
+# Micro-packaging
+
+Micro-packaging allows users to share Kedro pipelines across codebases, organisations and beyond.
+
+## Package a modular pipeline
+
+Since Kedro 0.16.4 you can package a modular pipeline by executing:
+
+`kedro pipeline package <pipeline_name>`
+
+* This will generate a new [wheel file](https://pythonwheels.com/) for this pipeline.
+* By default, the wheel file will be saved into `src/dist` directory inside your project.
+* You can customise the target with the `--destination` (`-d`) option.
+
+When you package your modular pipeline, Kedro will also automatically package files from 3 locations:
+
+```text
+├── conf
+│   └── base
+│       └── parameters
+│           └── {{pipeline_name*}}  <-- All parameter file(s) 
+└── src
+    ├── my_project
+    │   ├── __init__.py
+    │   └── pipelines
+    │       └── {{pipeline_name}}    <-- Pipeline folder
+    └── tests
+        ├── __init__.py
+        └── pipelines
+            └── {{pipeline_name}}    <-- Pipeline tests
+```
+
+Kedro will also include any requirements found in `src/<python_package>/pipelines/<pipeline_name>/requirements.txt` in the modular pipeline wheel file. These requirements will later be taken into account when pulling a pipeline via `kedro pipeline pull`.
+
+```eval_rst
+.. note::  Kedro will not package the catalog config files even if those are present in ``conf/<env>/catalog/<pipeline_name>.yml``.
+```
+
+If you plan to publish your packaged modular pipeline to some Python package repository like [PyPI](https://pypi.org/), you need to make sure that your modular pipeline name doesn't clash with any of the existing packages in that repository. However, there is no need to rename any of your source files if that is the case. Simply alias your package with a new name by running `kedro pipeline package --alias <new_package_name> <pipeline_name>`.
+
+In addition to [PyPI](https://pypi.org/), you can also share the packaged wheel file directly, or via a cloud storage such as AWS S3.
+
+## Package multiple modular pipelines
+
+To package multiple modular pipelines in bulk, run `kedro pipeline package --all`. This will package all pipelines specified in the `tool.kedro.pipeline.package` manifest section of the project's `pyproject.toml` file:
+
+```toml
+[tool.kedro.pipeline.package]
+first_pipeline = {alias = "aliased_pipeline", destination = "somewhere/else", env = "uat"}
+second_pipeline = {}
+```
+
+* The keys (`first_pipeline`, `second_pipeline`) are the names of the modular pipeline folders within the codebase.
+* The values are the options accepted by the  `kedro pipeline package <pipeline_name>` CLI command.
+
+```eval_rst
+.. note::  Make sure `destination` is specified as a POSIX path even when working on a Windows machine.
```

Comment:
Great note! 👌

Reply:
Not mine just stolen from the old stuff 😂